### DISCUSSION TITLE
devTools opens on any `Kit` > `Hot 🔥` item

### DISCUSSION THREAD

**Discussion by antenando** [johnlindquist/kit#841] [Errors] [answered]
Not sure if this is an error but...

...if I go to to `Kit` > `Hot 🔥` and choose any _hot_ item, it will open my default browser on the chosen page AND a developer tools window with `~/.kenv/tmp/devTools/devTools.html` ?

Is this the correct behaviour or something is broken on my end? I wasn't expect the devTools to open... ¯\_(ツ)_/¯

**Comment by johnlindquist** [comment, marked as answer]
🤦‍♂️ my bad, thanks for letting me know. I left in a 'dev()' call.

If you want to patch it, open:

~/.kit/main/hot.js

And remove the 'dev(url)' line. 

I'll get this sorted in the next release

**Reply by antenando** [reply]
No worries, John! Thanks!
